### ISSUE TITLE
Show warning when "intended start date" lies within two weeks of today

### ISSUE THREAD

**Issue by miggol** [DH-IT-Portal-Development/ethics#352]
In such cases, the FETC won't be able to provide official approval, and submitters should be warned before submission. We should show a warning on the submission page and on the first page where the field is located.

The warning on the submission page can easily be made dependant on user input. However, a warning on the same page as the field might be a hassle to make dynamically, and so a permanent warning would be fine there too.

**Comment by tymees** [comment]
> The warning on the submission page can easily be made dependant on user input. However, a warning on the same page as the field might be a hassle to make dynamically, and so a permanent warning would be fine there too.

My first thought would be 'why not just add a validiation step in clean', but ofc that would block submission...

But I think making a bit more effort to make the warning on the same page dynamic is worth the investment. A static warning will be ignored unless we make it visually 'scary', but that would cause undue stress for those who the warning does not apply.

The WMO page already does something similar, we might be able to hook into that?

**Comment by djhcapel** [comment]
Could this issue also get priority? It will save both the secretary and the applicant (a lot of) time if applicants are warned before they actually submit their application. It does happen regularly that applicants fill in a date that lies in the past or forget to update the 'intended start' date during revision.

**Comment by miggol** [comment]
I've started work on this. Submission page now shows the appropriate warning.

> But I think making a bit more effort to make the warning on the same page dynamic is worth the investment. 

From my call with Desiree about this issue, which admittedly was a while back, I think we concluded this information should always be visible. Most submitters never read the FETC guidelines, and this tidbit is important enough to always emphasize. This is generally educational, but also important for submitters who first create their proposal with plenty of time, work on it for a few weeks, then finally submit after the deadline. They would have wanted to know about the two weeks from the start!

> A static warning will be ignored unless we make it visually 'scary', but that would cause undue stress for those who the warning does not apply.

This is a good point. I will look into adding a dynamic visual emphasis to the message if the deadline is close or has passed.

> The WMO page already does something similar, we might be able to hook into that?

That's a good tip, I will be basing whatever it becomes off of that!
